merge/merger.go

```diff
@@ -30,8 +31,77 @@ func NewMerger(fhirHost string) *Merger {
 // is successful a new FHIR Bundle containing the merged patient record is returned.
 // If a merge fails, a FHIR Bundle containing one or more OperationOutcomes is
 // returned detailing the merge conflicts.
-func (m *Merger) Merge(source1, source2 string) (mergeID string, outcome *models.Bundle, err error) {
-	return mockMerge(source1, source2)
+func (m *Merger) Merge(source1ID, source2ID string) (mergeID string, outcome *models.Bundle, err error) {
```

Comment:
The reason I renamed the arguments in the header to "source1" and "source2" is that they are fully qualified URLs pointing to bundles, not just the IDs. I'd prefer we keep it that way.